docs/framework/20categories/20architecture/messaging.md

```diff
@@ -37,6 +37,19 @@ While the above approaches offer better security guarantees because they remove
   * What is the cost of bribing or corrupting a threshold of such validators to violate safety or liveness?
   * How does the bridge adapt to active misbehavior by a portion of the validators?
 
-* Optimistic approaches - honest minority (but incentive alignment that such a minority exists)
+* Optimistic approaches: rely on agents that validate cross-chain transactions by signing a merkle root with data from the source network and post it on the destination network. Once this data is posted, a challenge window begins, during which anyone monitoring the bridging system can challenge a fraudulent transaction by submitting fraud proofs on the source network and prevent it from being confirmed on the destination network. Such bridges assume that a) agents are incentivized to sign only legitimate transactions because their bonded funds will be slashed if they act maliciously and b) that in the event an agent does sign a fraudulent transaction, one honest actor will be watching the system, and they will flag the fraudulent transaction by submitting a fraud-proof on the source network within the challenge window. Thus, optimistic bridges have a 1 of N security model, which relies on one honest actor watching the system to correctly verify cross-chain transactions.
 
-A number of protocols employ a hybrid approach, that utilizes different approaches for different legs of the cross-chain interaction.
+  Considerations:
+  
+  * How many agents are employed by the bridge to sign and validate transactions? Is this set of agents centralized? If so, can the agents conduct a Denial-of-Service (DoS) attack by not signing a merkle root? In such cases, will the system halt?
```

Comment:
I think another question is how watchers are incentivized. Watchers have to do a fair bit of work to ensure safety and might only get rewarded on the off chance that fraud is attempted. So it might be worth asking do we assume watchers to a) be altruistic actors, doing this for the good of the protocol and its ecosystem, b) entities that are directly financially incentivized who expect to be rewarded in the event of fraud or some other condition, c) entities or applications that are indirectly incentivized to ensure the safety of the protocol (e.g., cross-chain applications that rely on the bridge) d) the bridge operators themselves 

d and a are probably not the most sustainable models, whereas b and c require a fair few considerations (e.g., verifiers dilemma). An associated consideration might be whether watcher fraud reports can be front-run, and the implications of that to watcher incentivization (e.g., Rainbow bridge)